war/src/main/webapp/scripts/sortable.js

```diff
@@ -52,7 +52,8 @@ var Sortable = (function() {
 
         var firstRow = this.getFirstRow();
         if (!firstRow) return;
-
+        // Active the filter
+        document.getElementById("filter-box").removeAttribute("disabled");
```

Comment:
I'm not convinced by this, given this javascript is applied to all sortable tabled, and not only the ones in the Plugin Manager. At least it should check if the element actually exists.

Reply:
I can confirm. I have manually tested it and it throws an error when loading the "All jobs" table on the home page:
`sortable.js:56 Uncaught TypeError: Cannot read property 'removeAttribute' of null`